Add blurhash image using unlazy.

dessalines opened the pull request:
This uses the library [unlazy](https://unlazy.byjohann.dev/guide/) to load load lazy placeholders from blurhash strings returned by the `postView.image_details` response.

~~I've tried several libraries, including [fast-blurhash](https://github.com/mad-gooze/fast-blurhash?tab=readme-ov-file), and client-side, they all take at least 500ms to convert a blurhash string to the base64 image string.~~

~~@asonix looked into mastodon, and found that they're likely doing it client-side also, which means they suffer from the same problem.~~ 

~~We can either accept that cost, or a better solution might be :~~ 

~~- Since we already store the `blurhash` string in our `image_details` table, then we could just calculate the base64 in the rust back-end, save it to the `image_details` table, then return it directly.~~ 

~~If anyone else thinks that's a good idea, then I'll open up an issue in lemmy's back-end for that. If its not worth it, then this is as good as we can do.~~

Fixes #3523 

~~EDIT: changed my mind, going to add the base64 saving into the lemmy back-end. Now front ends won't have to do this work.~~

https://github.com/LemmyNet/lemmy/pull/6091

# Example:

https://github.com/user-attachments/assets/401d8efc-88ed-4b51-9baf-e1698d649dfa

dessalines commented on src/shared/components/app/app.tsx:
```diff
@@ -1,3 +1,5 @@
+// @ts-expect-error has a weird import error
+import { lazyLoad } from "unlazy";
```
See https://github.com/johannschopplich/unlazy/issues/66

dessalines commented:
Turns out I didn't need to save a `base64` in the back-end. 

I found a workaround for the [slow blurhash computation](https://github.com/johannschopplich/unlazy/issues/67) issue.

dessalines commented:
When using the `data-blurhash` / client side rendering approach, I get this:

https://github.com/user-attachments/assets/29c74d02-8cba-4895-8ab5-d82bfb8545e9

I can verify that it is actually loading the blurhash (by temporarily removing data-src), but as you can see we get a weird image not loaded picture when not using the `src` attribute directly.

dessalines commented:
Using a tiny base64 placeholder for src seems to fix the error image showing:


https://github.com/user-attachments/assets/bc1f8693-3c54-4093-80f2-f60619a323b5

matc-pub commented:
> we get a weird image not loaded picture when not using the src attribute directly

I believe this is connected to eruda, with eruda in Firefox I get this warning:

```
Layout was forced before the page was fully loaded. If stylesheets are not yet loaded this may cause a flash of unstyled content.
```
The  browser has to figure what to display for those images too early. Eruda probably triggers this, when placing the dev tools button, by reading the current client size.

dessalines commented:
I tried commenting out eruda (and removing the base64 svg hack I added), but it still shows the same problem. Anyways the hack I added seems to work fine.
